_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,77 @@
+---
+name: image accessible name contains image file name
+description: |
+  This rule checks that the use of an image's file name in it's accessible name, is appropriate.
+
+success_criterion:
+- 1.1.1 # Non-Text Content
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Bryn Anderson
+---
+
+## Test procedure
+
+### Applicability
+The rule applies to any HTML `img` element that is exposed to assistive technologies, where the accessible name of the element contains the file name specified in the `src` attribute of the element.
```

Comment:
Bunch of thoughts:

- Link "exposed to assistive technologies". 
- Check the `alt` attribute, not the accessible name. Authoring tools do this to alt attributes, not to other stuff.
- Add an example of what the file name is. So: src="foo/bar.jpg?baz" has the file name `bar.jpg`.
- Add another condition that checks that the src and the alt attribute aren't the same
- IMO, this should look at an exact match between alt and filename, not if it is contained.